addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1053,16 +1080,27 @@ options.registry.OptionsTab = options.Class.extend({
             // comparaison with BS grays. However the system supports any
             // gray palette so we base our hue and extra saturation displayed
             // values on one gray, let's say the middle one: gray-500.
-            const gray = weUtils.getCSSVariableValue('500');
-            const grayRGB = ColorpickerWidget.convertCSSColorToRgba(gray);
-            const hsl = ColorpickerWidget.convertRgbToHsl(grayRGB.red, grayRGB.green, grayRGB.blue);
+            if (params.param === this.GRAY_PARAM_NAMES.EXTRA_SATURATION) {
+                const graysData = _.mapObject({max: params.idMax, min: params.idMin}, (val) => {
+                    const gray = weUtils.getCSSVariableValue(`${val}`);
```

Comment:
@qsm-odoo 
Here i have two choice 
- Fetch the values from params and convert them (max, min) in this case I have to reverse the construction of max and min (strong coupling)
- Extract saturation from the hex color that is in the css file.

**I chose the second, it's not complicated and not coupled to another treatment** 
What do you think?